main.tf

```diff
@@ -1,6 +1,41 @@
+locals {
+  dlq_options = length(var.dlq_options) > 0 ? defaults(var.dlq_options, {
+    visibility_timeout_seconds = 30
+    message_retention_seconds  = 1209600
+    name = format("%s_%s",var.name,"dlq")
+  }) : {}
+}
+
+module "sqs_queue_dlq" {
+  source                            = "terraform-aws-modules/sqs/aws"
+  version                           = "~> 3.2.0"
+  create                            = length(local.dlq_options) > 0 ? true : false
+  name                              = lookup(local.dlq_options, "name", "${var.name}_dlq")
+  fifo_queue                        = var.fifo_queue
+  visibility_timeout_seconds        = lookup(local.dlq_options, "visibility_timeout_seconds", 30)
+  message_retention_seconds         = lookup(local.dlq_options, "message_retention_seconds", null)
```

Comment:
This should be 14days in seconds for default

Reply:
Added